server/server.go

```diff
@@ -95,6 +95,7 @@ type Server struct {
 
 	EnableRegistration       bool
 	EnableClientRegistration bool
+	EnableClientUserCreation bool
```

Comment:
```
// If enabled, causes the API to authenticate using client credentials instead of an ID token.
// Clients must be "admin clients" to use the API.
EnableClientCredentialAccess
```